universal-application-tool-0.0.1/conf/messages

```diff
@@ -16,6 +16,9 @@
 # The text on the button an applicant clicks to log out of their session.
 button.logout=Logout
 
+# The text on the button an applicant clicks to jump to the review page.
+button.review=Review
```

Comment:
(blocking) Please add translations for all of these to messages.en-us and messages.es-us (I just use Google Translate for Spanish since the city said they generally use that).

Reply:
added